Cargo.toml

```diff
@@ -36,14 +36,15 @@ bincode = "1"
 criterion = { version = "0.4.0", features = ["html_reports"] }
 hex = "0.4.2"
 rand = "0.8"
+rand_core = { version = "0.6", default-features = false }
```

Comment:
I suppose not being default is justified by saying most dependencies already specify their own usages of the lower level methods.